<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Final Project - Goblet of SMOKE</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QDockWidget" name="settingsDock">
   <property name="minimumSize">
    <size>
     <width>147</width>
     <height>211</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="settingsDockContent">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="settingsLayout">
     <item>
      <widget class="QGroupBox" name="shaderGroupBox">
       <property name="title">
        <string>Shader program</string>
       </property>
       <widget class="QRadioButton" name="solidShader">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>140</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Solid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="textureShader">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>140</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Texture</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="gradientShader">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>140</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Gradient</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
